docs/debugging-openshift.md

```diff
@@ -57,6 +57,11 @@ The most recent container in that list should be the one that ran your build.  T
         
 Hopefully the logs will provide some indication of what it failed (e.g. failure to find the source repository, an actual build issue, failure to push the resulting image to the docker registry, etc).
 
+One issue see somewhat often is with not being able to resolve github.com in the build.  If this shows up in your build logs, restart docker and then resubmit a build:
```

Comment:
it's really not being able to resolve any hostnames from within running containers.